doc/sphinx/language/coq-library.rst

```diff
@@ -283,17 +284,17 @@ defined as inductive constructions over the sort ``Set``. Some of
 them come with a special syntax shown below (this syntax table is common with
 the next section :ref:`specification`):
 
-.. productionlist::
-   specif :  `specif` * `specif`                           (prod)
-          : `specif` + `specif`                          (sum)
-          : `specif` + { `specif` }                      (sumor)
-          : { `specif` } + { `specif` }                  (sumbool)
-          : { `ident` : `specif` | `form` }              (sig)
-          : { `ident` : `specif` | `form` & `form` }       (sig2)
-          : { `ident` : `specif` & `specif` }             (sigT)
-          : { `ident` : `specif` & `specif` & `specif` }    (sigT2)
-  term : (`term`, `term`)                               (pair)
-
+======================== ============================================= =======
+:production:`specif` ::= :n:`@specif * @specif`                        prod
```

Comment:
Same remark.

Reply:
Except this one has entries for both `specif` and `term`.  Please take a look at what I did here.